lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -44,6 +44,12 @@
     <% mode_class = '' %>
   % endif
 <div class="course-container">
+ % if course_program_info and course_program_info['category'].lower()=='xseries':
```

Comment:
see my comment above about checking the original category (instead of doing `lower()`)